examples/phi2/generate.py

```diff
@@ -0,0 +1,233 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import numpy as np
+import onnxruntime as ort
+import torch
+from transformers import AutoTokenizer
+
+pt_to_np = {
+    "torch.int32": np.int32,
+    "torch.int64": np.int64,
+    "torch.float32": np.float32,
+    "torch.float16": np.float16,
+}
+
+
+class ORTGenerator:
+    def __init__(self, decoder_path):
+        self.onnx_decoder_path = decoder_path
+        self.num_heads = 32
+        self.head_size = 80
+        self.num_layers = 32
+        self.max_sequence_length = 2048
+
+        self.use_fp16 = False
+        self.device = -1
+        self.use_buffer_share = False
+        self.packed_kv = False
+        self.use_step = False
+        self.torch_dtype = None
+        self.sess = None
+        self.tokenizer = None
+
+    def get_initial_inputs_and_outputs(self, encodings_dict):
```

Comment:
private method?

Reply:
The file is copied from ort. https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/transformers/models/phi2/inference_example.py#L91

We may can keep what it is now.

Reply:
Please add comments about where the file is copied.